### PR TITLE
Moved docs link to an Angular directive.

### PR DESCRIPTION
**Pull request by jgeewax**
- Moved the link to be next to the method
- Made the part that generates the link itself a directive
- Added the link next to classes as well

### CODE REVIEW

**Comment by ryanseys** [comment]
Tried locally and looks like it works! Thanks for making it more Angular-ly.

nitpick: Github is spelled GitHub (note the capitalization).

I'm also thinking maybe just `(view code [github icon image])` might look better? But this is an improvement nonetheless.

File: docs/site/components/docs/docs-directives.js [hunk 1]
```diff
@@ -9,4 +9,18 @@ angular.module('gcloud.docs')
...
+  .directive('docsCodeLink', function($compile) {
+    'use strict';
+
+    var GITHUB_BASE = 'https://github.com/GoogleCloudPlatform/gcloud-node/blob/';
+
+    return {
+      template: '<a href="' + GITHUB_BASE + '{{version}}/lib/' +
+                '{{module ? module + \'/\' : \'\'}}{{class}}.js' +
+                '{{method.lineNumLink}}">' +
+                '(code on Github)' +
```
Comment (ryanseys) [inline docs/site/components/docs/docs-directives.js]: GitHub

Comment (jgeewax) [inline docs/site/components/docs/docs-directives.js]: Fixed.

**Comment by jgeewax** [comment]
@ryanseys Does Travis usually take this long to build...?

**Comment by ryanseys** [comment]
Yep. Not our fault, it hasn't been assigned a server yet. It sucks!

File: docs/site/components/docs/docs.html [hunk 2]
```diff
@@ -128,6 +129,8 @@ <h2 ng-if="method.name[0].toUpperCase() === method.name[0]">
           class="permalink"
           ng-href="{{activeUrl + '?method=' + method.name}}">#</a>
         {{method.name}}
+        <span class="view-code-link" docs-code-link></span>
+        </span>
```
Comment (ryanseys) [inline docs/site/components/docs/docs.html]: Oops missed this in my first pass. This is an extra closing `span` tag.

Comment (jgeewax) [inline docs/site/components/docs/docs.html]: Fixed.

**Comment by ryanseys** [comment]
Thanks!

**Pull request merged**